File: leetcode/array/1313.decompress-run-length-encoded-list.py
class Solution:
    def decompressRLElist(self, nums: List[int]) -> List[int]:
        if not nums or len(nums) % 2 != 0:
            return

        solution = []

        for i in range(1, len(nums), 2):
            solution += [nums[i]] * nums[i-1]

        return solution

File: leetcodeJS/array/1313.decompress-run-length-encoded-list.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var decompressRLElist = function (nums) {
  let solution = [];
  for (let i = 1; i < nums.length; i += 2) {
    j = nums[i - 1];
    while (j > 0) {
      solution.push(nums[i]);
      j--;
    }
  }
  return solution;
};
